doc/internal/man3/OSSL_TIME.pod

```diff
@@ -2,19 +2,23 @@
 
 =head1 NAME
 
-OSSL_TIME, OSSL_TIME_SECOND, OSSL_TIME_INFINITY,
-ossl_time_now, ossl_time_time_to_timeval, ossl_time_compare,
+OSSL_TIME, OSSL_TIME_SECOND, OSSL_TIME_INFINITY, OSSL_TIME_IMMEDIATE,
+ossl_time, ossl_time_now, ossl_time_time_to_timeval, ossl_time_compare,
 ossl_time_add, ossl_time_subtract
 - times and durations
 
 =head1 SYNOPSIS
 
  #include "internal/time.h"
 
- typedef uint64_t OSSL_TIME;
+ typedef struct OSSL_TIME;
 
- #define OSSL_TIME_SECOND
- #define OSSL_TIME_INFINITY
+ #define OSSL_TIME_SECOND   /* Ticks per second */
+
+ OSSL_TIME OSSL_TIME_INFINITY;
+ OSSL_TIME OSSL_TIME_IMMEDIATE;
```

Comment:
This re-introduces global variables, which find-doc-nits currently doesn't understand.  Hmm...

Reply:
They should be also const/

Reply:
I've made them inline functions.  This avoids the const question and the globals.